Sources/SPMTestSupport/MockWorkspace.swift

```diff
@@ -61,48 +59,48 @@ public final class TestWorkspace {
         self.skipUpdate = skipUpdate
         self.enablePubGrub = enablePubGrub
 
-        try create()
+        try self.create()
```

Comment:
These were automatically inserted by SwiftFormat, for which @tomerd added the configuration with #3012. I believe  we share stylistic preferences, but I'm more than happy to accede to any conventions enforced by a formatter.

That said, I've encountered a few situations in which SwiftFormat makes source-breaking changes to Swift Package Manager. Namely, mistakenly inserting `self.` for things like `break` labels and local variables whose names shadow an instance method or property. I'm working to share minimal, reproducible examples with the maintainer, but this may be cause for reconsidering how automatic formatting is currently set up.

Reply:
> Are all these self. additions just stylistic? I think the general Swift approach (and the one used in SwiftPM) is to not specify self. when it's not needed (such as in an escaping closure).

> These were automatically inserted by SwiftFormat, for which @tomerd added the configuration with #3012. I believe we share stylistic preferences, but I'm more than happy to accede to any conventions enforced by a formatter.

In other projects (SwiftNIO, etc) we actually encouraged adding self. to make sure things are very explicit. happy to open up for discussion tho. cc @weissi

> That said, I've encountered a few situations in which SwiftFormat makes source-breaking changes to Swift Package Manager. Namely, mistakenly inserting self. for things like break labels and local variables whose names shadow an instance method or property. 

interesting. I have not had such trouble with SwiftFormat in other projects, what version are you using @mattt?

Reply:
@mattt in any case dont let this block progress. you can run the formatter with the `self` setting if it breaks things and we can revisit this separately

Reply:
@tomerd According to `swiftformat --version`, I'm using the latest (0.47.2). For now, I'm committing with `--no-verify` to skip my pre-commit hook as necessary. So this isn't a blocker.

Reply:
This is fine, it's just opposite of what I've seen before so I wanted to ask.